### ISSUE TITLE
Rule to disallow skipped tests

### ISSUE THREAD

**Issue by m-burst** [m-burst/flake8-pytest-style#30]
# Rule request

## Description

Disallow unconditionally skipped tests.

Examples of bad code:
```py
@pytest.mark.skip(reason='some reason')
def test_skipped():
    ...
```

```py
def test_skipped():
    pytest.skip('some reason')
```

NB: this should **not** be reported as a violation:
```py
def test_conditionally_skipped():
    if some_condition:
        pytest.skip('some reason')
```

## Rationale

* to make sure skipped tests are not accidentally committed
* inspired by [eslint-plugin-jest rule `no-disabled-tests`](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-disabled-tests.md)

**Comment by sobolevn** [comment]
What about `skipif(True)`?

**Comment by m-burst** [comment]
Thanks @sobolevn, I've added it to the description
